### PR TITLE
HHH-13133

### PR DESCRIPTION
**Pull request by skis**
Pull request to HHH-13133

### CODE REVIEW

File: tooling/hibernate-enhance-maven-plugin/src/main/java/org/hibernate/orm/tooling/maven/MavenEnhancePlugin.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public boolean doExtendedEnhancement(UnloadedClass classDescriptor) {
 
 			writeOutEnhancedClass( enhancedBytecode, file );
 
-			getLog().info( "Successfully enhanced class [" + file + "]" );
```
Comment (Sanne) [inline tooling/hibernate-enhance-maven-plugin/src/main/java/org/hibernate/orm/tooling/maven/MavenEnhancePlugin.java]: could you change this to make sure the strings aren't appended when the logger is not set to debug?

Something like

    log = getLog()
   if (log.isDebugEnabled()) { log.debug( .... ) }

**Comment by Sanne** [comment]
Many thanks! Only had a very minor comment, if you could address that I'll merge it right away.

Generally speaking, try to not include unrelated changes in the same commit / same JIRA id.. but it's fine since they are very minor.

**Comment by Sanne** [comment]
merged it. Thanks!

**Comment by skis** [comment]
Thanks for merging Sanne. Strangely I didn't get any notification for the last 4 days.

File: tooling/hibernate-enhance-maven-plugin/src/main/java/org/hibernate/orm/tooling/maven/MavenEnhancePlugin.java (hunk 1 above)
Comment (skis) [reply tooling/hibernate-enhance-maven-plugin/src/main/java/org/hibernate/orm/tooling/maven/MavenEnhancePlugin.java]: I will make the change and submit another PR.

**Comment by Sanne** [comment]
@skis no worries, and no need to make another PR as I already changed it!

Thanks again

**Pull request closed without merging**